Casks/i/ibkr.rb

```diff
@@ -0,0 +1,41 @@
+cask "ibkr" do
+  arch arm: "arm", intel: "x64"
+  os = on_arch_conditional arm: "macos", intel: "macosx"
```

Comment:
I feel like this could be simplified to
```suggestion
  arch arm: "-arm", intel: "x-x64"
```
See url comment.